README.md

```diff
@@ -58,37 +58,38 @@ data = sdr.rx()
 
 ### Installing from source
 ```
-tcollins@jeeves:~$ git clone https://github.com/analogdevicesinc/pyadi-iio.git
-tcollins@jeeves:~$ cd pyadi-iio
-tcollins@jeeves:~$ (sudo) pip install .
+git clone https://github.com/analogdevicesinc/pyadi-iio.git
+cd pyadi-iio
+(sudo) python3 -m pip install pip -U
+(sudo) pip install .
```

Comment:
This is a little inconsistent. To match other other pip commands I might just remove the python3 -m prefix